docs/state.md

```diff
@@ -435,8 +435,14 @@ WindowStore
       }
     },
     hasBitcoinHandler: boolean,  // brave browser has a `bitcoin:` URI handler
-    paymentIMG: string,           // the QR code equivalent of `paymentURL` expressed as "data:image/...;base64,..."
-    error: { // non-null if the last updateLedgerInfo happened concurrently with an error
+    countryCode:                 // ISO3166 2-letter code for country of browser's location
+    exchangeInfo: {              // information about corresponding "friendliest" BTC exchange (suggestions welcome!)
+      titleText: string,         // e.g., 'Create an account on ...'
+      buttonText: string,        // e.g., 'Create'
+      buttonURL: string,         // where the button should take the browser      
```

Comment:
looks like https://ledger.brave.com/v1/exchange/providers is not localized, so take out 'buttonText' and 'titleText' and just add a property for the name of the exchange.